README.md

```diff
@@ -1,2 +1,41 @@
-# hw01
-Repository for first CFSS homework assignment
+#Computing for Social Sciences, Homework 01
+
+My name is **Poh Yu Quan**. My *last name* is **Poh** and my first name is **Yu Quan**. I am a **MAPSS student** at the University of Chicago. I come from **Singapore** but I obtained my BSc at **Imperial College London** in *Biomedical Sciences*. I am concentrating in sociology and focusing on criminology.
+
+My research interests include:
+
+1. Criminology
+2. Urban Sociology
+
+My hobbies include:
+
+1. Rock Climbing
+  * Bouldering
+  * Sport Climbing
+2. Snowboarding
+3. Hiking
+4. Travelling
+5. Programming
+  * Python
+  * R
+  * SQL
+
+Here is a handsome picture of me for your viewing pleasure. Please **don't photoshop** it. :smiley:
+
+![My Handsome Face](http://i1174.photobucket.com/albums/r603/Poh_Yu_Quan/14717174_10153835225981746_9155896163653390504_n_zpsgckh02iw.jpg)
+
+Also you can visit my GitHub [here](https://github.com/pohyuquan).
+
+##How I got my changes onto GitHub:
+
+1. First I **forked** the HW repository.
+2. Next I **cloned** the forked repository onto my computer using RStudio.
+  * Created a new project in RStudio using version control (Git) and pasted the URL for my forked repository.
+3. I modified the *README.md* file in RStudio using Markdown Syntax as indicated by Github (found [here](https://guides.github.com/features/mastering-markdown/)).
+4. I committed my changes and added a message "modified README" to indicate my changes.
+5. I pulled from GitHub using the "Git" tab in RStudio to update my local repository.
+6. I pushed my changes to GitHub through RStudio in the same "Git" tab as before.
+7. I made a pull request to merge the branch with the master.
+
+##Reflections
+The process of cloning a repository and making local changes before committing it and pushing it was fairly straight forward as I referenced the steps indicated through the CFSS "General Homework Guidelines" page (found [here](http://cfss.uchicago.edu/hw00_homework_guidelines.html#homework_workflow)). Although I had previously used a different command interface (*cygwin*), it took some getting used to in order to learn how to perform the similar commands on RStudio. 
```

Comment:
Evaluation HW-1 for Poh Yu Quan
----------------------------------------------------
| Topic                                 | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**                   |    x       |              |            |
| **Coding strategy**             |     x      |              |            |
| **Presentation**                  |    x        |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**               |     x      |              |            |

Remarks:

* Overall a good use of all of the required Markdown code. It was well integrated into your biography.
* I like the use of the picture, it added a lot to the visual diversity of your page?
* I would suggest adding a bit more to your reflections and commits. I think detailing out some more of your decision making process would be helpful in understanding and replication.